hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OmMetadataManagerImpl.java

```diff
@@ -318,6 +318,24 @@ public class OmMetadataManagerImpl implements OMMetadataManager,
   private final Map<String, TableCacheMetrics> tableCacheMetricsMap =
       new HashMap<>();
   private SnapshotChainManager snapshotChainManager;
+  private final S3Batcher s3Batcher = new S3Batcher() {
```

Comment:
nit: How about having this object inside a constructor keeping this in a method like initializeS3Batcher()?

Reply:
There are 4 constructors.  What is the benefit of duplicating the assignment in all of them?

Reply:
Converted the anonymous class to an inner one.